djamazing/storage.py

```diff
@@ -72,15 +74,16 @@ def __iter__(self):
 class DjamazingStorage(Storage):
 
     def __init__(self, config=None):
-        config = config or settings.DJAMAZING
-        self.cloud_front_base_url = config['CLOUDFRONT_URL']
+        self.config = DEFAULT_SETTINGS
```

Comment:
I'm wondering if it's better to create a copy of the default settings dict just to ensure it's safe in all edge cases (and thread safe). Update in next line changes defaults forever.